src/table/paginator.scss

```diff
@@ -78,3 +47,38 @@
     text-overflow: ellipsis;
   }
 }
+
+@mixin iui-paginator-ellipsis {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $iui-component-height;
+  @include themed {
+    color: t(iui-text-color-muted);
+  }
+
+  &.iui-small {
+    width: $iui-component-height-small;
+  }
+}
+
+@mixin iui-paginator-pages-group {
+  margin: 0 $iui-s;
+
+  > * {
+    margin-right: 0; // unset negative margin from button-group
+  }
+
+  .iui-paginator-page-button {
+    width: $iui-component-height;
+
+    &.iui-small {
+      width: $iui-component-height-small;
```

Comment:
This approach ties the page-button styles to our base button classes (which might change in the future). It might be better to add a new modifier here (and include the button-sizes mixin to inherit any base styles).

Reply:
amm.. so you want me to do same for small and active modifiers? Go like `iui-paginator-button-small` or something like that?

Reply:
Yeah something like that. It might even make sense to make the `-small` a modifier on the paginator root, because it would allow decoupling the paginator height from the table density

Reply:
done